Orange/widgets/visualize/owlinearprojection.py

```diff
@@ -329,7 +329,8 @@ def effective_variables(self):
 
     @property
     def effective_data(self):
-        return self.data.transform(Domain(self.effective_variables))
+        return self.data.transform(Domain(self.effective_variables,
+                                          self.data.domain.class_vars))
```

Comment:
I just realised this wont work because class variable is duplicated when selected in effective variables.